modules/thunder_search/config/optional/views.view.media_search_api.yml

```diff
@@ -0,0 +1,887 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_expires
+    - image.style.thumbnail
+    - search_api.index.content
+  module:
+    - datetime
+    - image
+    - search_api
+    - thunder_search
+    - user
+    - views_bulk_operations
+    - config_selector
+id: media_search_api
+label: Media
```

Comment:
Label for Media view with search API should be adjusted or at least Description. Since in Configuration Selector there are 2 views with same name and no description. :(